launcher/internal/experiments/experiments_test.go

```diff
@@ -0,0 +1,51 @@
+package experiments
+
+import (
+	"testing"
+)
+
+func TestExperiments(t *testing.T) {
+	tests := []struct {
+		input string
+	}{
+		{input: "{\"EnableTestFeatureForImage\":true,\"EnableSignedContainerImage\":true}"},
```

Comment:
Add a test for invalid JSON (e.g., missing a closing bracket).

Reply:
This is in the next test case.